modules/KalturaSupport/UiConfResult.php

```diff
@@ -240,6 +243,7 @@ private function walkThroughPluginAttributes( $attrArray ,&$refrencePathInObject
 				$this->walkThroughPluginAttributes( $attrValue , $refrencePathInObject[$attrKey] );
 			}
 			else {
+			    if( in_array($attrKey, $this->propsToIgnoreWhenResolvingResource) ) continue;

```

Comment:
@itaykinnrot - when we DO use {html5ps} it doesn't work, with this small fix it does. I don't have Ran now, but I don't want to miss this merge. I think that specifically for templatePath it doesn't work (while other paths do)

Can you or someone from your team review this? (and make sure that this didn't break core templatPath) 
Client is an important one and was promised this for a while.